### PR TITLE
Add more profiling payloads

### PR DESCRIPTION
**Pull request by mre**
I kept it super simple for now. Just duplicated the profiling code for now. We could go fancy and use traits and a builder pattern, but what for? It's just a poor little profiling tool. 😉

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 SHELL := /bin/bash
 .PHONY: help
 
-ts := $(shell date --utc +%FT%TZ)
+ts := $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
```
Comment (konstin) [inline Makefile]: You might like `date --iso-8601=seconds` or `date -u --iso-8601=seconds`

Comment (mre) [reply Makefile]: Doesn't work on macOS. 😢 
```date --iso-8601=seconds
date: illegal option -- -
usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ...
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]```

Comment (konstin) [reply Makefile]: :disappointed:

File: profiling/src/profiles/booleans.rs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+use pyo3::prelude::*;
+
+pub fn exec(iterations: u64) {
+    let gil = Python::acquire_gil();
+    let py = gil.python();
+
+    let booleans = format!("[{} \"true\"]", "\"true\",".repeat(256));
```
Comment (mre) [inline profiling/src/profiles/booleans.rs]: Is there a more idiomatic way to create a JSON string with 256 booleans?
`["true","true","true","true",..."true"]`

Comment (konstin) [reply profiling/src/profiles/booleans.rs]: I'd actually know a way in java :sweat_smile:. For rust there's afaik no better way except maybe using serde on a Vec from `[true; 256]`, but imho the current code isn't bad.

Comment (konstin) [reply profiling/src/profiles/booleans.rs]: (btw it's 257 booleans. I'm curious if there's a measurable difference between 255, 256 and 257)

**Comment by mre** [comment]
Think we're good here. Thanks for the review @konstin. Merging.

**Pull request merged**